docs/getting-started/creating-your-first-project.md

```diff
@@ -31,12 +31,16 @@ You have a test project up and running! Visit `http://localhost:3000` and view y
 **Did you get a "Permission Denied" error?** On Linux, this would be because global npm packages must be installed as root in a typical setup. Use `sudo npm install` instead. On a Mac, this shouldn't happen if you installed node with Homebrew, because the `/usr/local` folder where Homebrew keeps its files should belong to your personal account. However, if you installed Homebrew with `sudo`, you'll need to be consistent and use `sudo` here too.
 :::
 
-**Important: ** `test-project`** is the name for our example. For your project use your own project's "shortname" containing only letters, digits, hyphens and/or underscores. It will be used by default as a MongoDB database name and a basis for cookie names, etc.** (Hyphens seem more popular than underscores for such purposes.) We'll continue to use `test-project` for the rest of this example. Alternatively, you can install a robust, themed, fully-baked project with this command:
+**Important: ** `test-project`** is the name for our example. For your project use your own project's "shortname" containing only letters, digits, hyphens and/or underscores. It will be used by default as a MongoDB database name and a basis for cookie names, etc.** (Hyphens seem more popular than underscores for such purposes.) We'll continue to use `test-project` for the rest of this example.
+
+Alternatively, you can install a robust, themed, fully-baked project with this command:
 
 ```bash
 apos create-project --boilerplate https://github.com/apostrophecms/apostrophe-open-museum test-project
 ```
 
+This will download a fully functional Apostrophe application codebase. Please note that while the Open Museum codebase does power [the official demo](http://demo.apostrophecms.org), the demo's uploaded assets and database are not included when using it as a boilerplate.
```

Comment:
This is great, how about adding:

"If you wish to see how these widgets and pieces and page types perform when content has already been added to the site, just try them out on the demo site."

Reply:
That seems overly explicit given the purpose of this section. The section is about creating your own project, not testing out Apostrophe. I'm going through various wordings, but they all seem beside the point of this tutorial.